packages/databricks-vscode/src/telemetry/constants.ts

```diff
@@ -10,13 +10,21 @@ export const DEV_APP_INSIGHTS_CONFIGURATION_STRING =
 /* eslint-disable @typescript-eslint/naming-convention */
 export enum Events {
     COMMAND_EXECUTION = "commandExecution",
-    EXTENSION_ACTIVATED = "extensionActivation",
+    EXTENSION_ACTIVATION = "extensionActivation",
+    EXTENSION_INITIALIZATION = "extensionInitialization",
+    AUTO_LOGIN = "autoLogin",
+    MANUAL_LOGIN = "manualLogin",
+    AUTO_MIGRATION = "autoMigration",
+    MANUAL_MIGRATION = "manualMigration",
+    BUNDLE_RUN = "bundleRun",
     CONNECTION_STATE_CHANGED = "connectionStateChanged",
-    SYNC_DESTINATION = "syncDestination",
-    SWITCH_TO_WORKSPACE_PROMPT = "switchToWorkspacePrompt",
 }
 /* eslint-enable @typescript-eslint/naming-convention */
 
+export type AutoLoginSource = "init" | "hostChange" | "targetChange";
+export type ManualLoginSource = "authTypeSwitch" | "authTypeLogin" | "command";
```

Comment:
Bundle init is another source

Reply:
Or manual migration. The manual login actions there are different though - they don't lead to logged-in extension state (and don't capture `ConnectionManager.connect` flow), that's why I didn't include them. We can add lower level event for LoginWizardExecution with more sources later.

But that reminds me that I forgot to add telemetry for the bundle-init flow at all, gonna add it